chpl/chpl-api/e2e/collections/product-version-controller.postman_collection.json

```diff
@@ -0,0 +1,144 @@
+{
+	"info": {
+		"name": "product-version-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "PUT /versions - merge request permission restriction validation - anonymous",
```

Comment:
I don't see these test running in bamboo. Is it pushed to dev?

Reply:
nope it is not yet pushed to DEV, I wanted to check out the tests from other PR that I pushed yesterday. I am going to push these out today

Reply:
Tests were pushed to dev, plan was run and tests ran as expected